src/main/java/ChatBot.java

```diff
@@ -0,0 +1,95 @@
+import java.time.LocalDate;
+import java.time.LocalTime;
+import java.util.Scanner;
+
+/**
+ * Contains the ChatBot called by Main class.
+ */
+public class ChatBot {
+
+    /** Indicates if ChatBot has been ended by User */
+    static boolean hasEnded = false;
+
+    /**
+     * Main static method for ChatBot to be started in Main class.
+     */
+    public static void start(){
+        try {
+            DataStorageInterface.initStorage();
+            Scanner scanner = new Scanner(System.in);
+            System.out.println("Hello I'm Duke!\nWhat can I do for you?");
+            while (!ChatBot.hasEnded) {
+                try {
+                    String query = scanner.nextLine();
+                    String resp = ChatBot.response(query);
+                    System.out.println(resp);
+                } catch (Exception e) {
+                    System.out.println(e.getMessage());
+                }
+            }
+        } catch(DukeException e){
+            System.out.println("ChatBot initialisation failed.");
+        }
+    }
+
+    /**
+     * Returns appropriate response to be printed on the screen for the User to read.
+     *
+     * @param query The String passed in by the User.
+     * @return Response to User's command.
+     * @throws Exception Encompasses DukeException, FileNotFoundException, IOException
+     */
+    private static String response(String query) throws Exception{
+        String[] splitQuery = Parser.getSplit(query);
+        String command = Parser.getCommand(splitQuery);
+        String[] cRemoved = Parser.removeCommandString(splitQuery);
+        switch (command){
+            case "bye":
```

Comment:
Perhaps remove indentation for switch case, as according to the module formatting guide?